spec/services/active_department_employees_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ActiveDepartmentEmployeesService do
+  context 'when there are no records' do
+    let(:result) { described_class.new.call }
+
+    it 'returns an empty array' do
+      expect(result).to be_empty
+    end
+  end
+
+  context 'when there are saved records' do
+    let!(:active_employments) { create(:employment, :active) }
+
+    it 'returns instances of employees objects' do
+      employees = described_class.new.call
+      expect(employees.first).to be_an_instance_of(Employee)
```

Comment:
```suggestion
      employees = service.call
      expect(employees.pluckl(:registration, :name, :department).to match_array([[active_employee.registration, active_employee.employee.name, active_employee.department.name]])
```

You can test the content of your query with something like this. Let me know you have questions.